runners/core-java/src/test/java/org/apache/beam/runners/core/metrics/MetricsContainerImplTest.java

```diff
@@ -178,6 +178,34 @@ public void testMonitoringInfosArePopulatedForUserCounters() {
     assertThat(actualMonitoringInfos, containsInAnyOrder(builder1.build(), builder2.build()));
   }
 
+  @Test
+  public void testMonitoringInfosArePopulatedForUserDistributions() {
+    MetricsContainerImpl testObject = new MetricsContainerImpl("step1");
+    DistributionCell c1 = testObject.getDistribution(MetricName.named("ns", "name1"));
+    DistributionCell c2 = testObject.getDistribution(MetricName.named("ns", "name2"));
+    c1.update(5L);
+    c2.update(4L);
+
+    SimpleMonitoringInfoBuilder builder1 = new SimpleMonitoringInfoBuilder();
+    builder1.setUrnForUserDistribution("ns", "name1");
+    builder1.setInt64DistributionValue(DistributionData.create(5, 1, 5, 5));
+    builder1.setPTransformLabel("step1");
+    builder1.build();
```

Comment:
No need to call build() here and next initialization. You do it at L206.
Also it is not required since you're ignoring return value.